notes/bugfix-120112.md

```diff
@@ -0,0 +1 @@
+# Unlock cursor on tab-command-alt key
```

Comment:
Thank you @BerndN

You have a typo at the name of the bugfix file, it should be `bugfix-20112.md` :)